pom.xml

```diff
@@ -25,8 +25,8 @@
         <guava.version>19.0</guava.version>
         <jackson.version>2.7.2</jackson.version>
         <azuresdk.version>1.3.0</azuresdk.version>
-        <azure-credentials.version>1.3.1</azure-credentials.version>
-        <azure-commons.version>0.1.5</azure-commons.version>
+        <azure-credentials.version>1.4.0</azure-credentials.version>
+        <azure-commons.version>0.2.2.2</azure-commons.version>
```

Comment:
0.2.3 fixed the common-langs3 issue

Reply:
sure, I will wait for the new `azure-commons` release.